base/docs/helpdb/Base.jl

```diff
@@ -1767,16 +1767,19 @@ deserialize
 """
     length(collection) -> Integer
 
-For ordered, indexable collections, returns the maximum index `i` for which `getindex(collection, i)`
-is valid.
-For unordered collections, returns the number of elements.
+Return the number of elements in collection.
```

Comment:
Yes, here it's not really the argument, it's just the English word, but if you understand it as the argument it also works, so I don't think there's a problem.